recipes/paratext/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd python/
+$PYTHON setup.py build install --prefix=$PREFIX
```

Comment:
Actually just noticed that this package is using `distutils`. I removed the build command and removed setuptools from the dependencies. Is there some flags that I should pass to distutils instead?